sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -111,6 +114,7 @@ export default {
     const isFirstPage = ref(true);
     const isLastPage = ref(false);
     let applicationWizard: any;
+    const isReadOnly = ref(false);
```

Comment:
I would recommend starting with true. It seems to be safer in the sense that the user will not be able to interact with the form till the data is loaded.